Code/Engine/Core/World/Implementation/World.cpp

```diff
@@ -548,6 +548,7 @@ void ezWorld::Update()
     EZ_PROFILE_SCOPE("Delete Dead Objects");
     DeleteDeadObjects();
     DeleteDeadComponents();
+    ProcessUpdateFunctionsToDeregister();
```

Comment:
I would maybe put this call directly before the ProcessUpdateFunctionsToRegister call. This place seems a bit odd.